docs/How-to-add-new-models.md

```diff
@@ -0,0 +1,28 @@
+# New OSS model onboarding
+
+This document describes how to add a new supported OSS model in Kaito. The process is designed to allow community users to initiate the request. Kaito maintainers will follow up and deal with managing the model images and guiding the code changes to set up the model preset configurations.
+
+## Step 1: Make a proposal
```

Comment:
Another format to consider but I like either/or: 


Step 1: Submit a Proposal:
* Initiator: Requestor (Community User)
* Action: Submit a PR using the specified template to describe the OSS model.
* Review: Kaito maintainers will review the PR for factors such as popularity, licensing, and model capabilities and decide whether to accept or reject the PR. 
* Status: Proposal status should be provisional.

Step 2: Validate and test the model:
* Initiator: Kaito Maintainers
* Action: We will download and test the model using the specified runtime, automated through GitHub actions.
* Documentation: Updates made to supported_models.yaml to include the new model.

Step 3: Push model image to MCR
Initiator: Kaito Maintainers
Action: If licensing permits, the model image is pushed to Microsoft Container Registry (MCR) and made publicly available. This step is skipped if the preset only allows private access.
Status Update: Proposal status changed to `ready to integrate`. 

Step 4: Add preset configurations
* Initiator: Requestor
* Action: Submit PR to contain code changes to implement a simple inference interface. [Here](../presets/models/falcon/model.go) is an existing example.
* Status Update: In the same or seperate PR, update the proposal status to integrated.

Step 5: Add an E2E test
* Initiator: Requestor
* Action: Add a new E2E test to ensure the inference service works as expected with the new model presets.